src/Loggable/Entity/MappedSuperclass/AbstractLogEntry.php

```diff
@@ -96,153 +105,80 @@ public function getId()
         return $this->id;
     }
 
-    /**
-     * Get action
-     *
-     * @return string|null
-     */
     public function getAction()
     {
         return $this->action;
     }
 
-    /**
-     * Set action
-     *
-     * @param string $action
```

Comment:
should we keep the `@param` info? in some places is kept, apparently PHPStan complains if it is not from level 6: https://phpstan.org/r/e1a70fb8-4b4d-48c4-bb16-7fd65e2e5d5a

Reply:
Thank you for pointing.
I reverted the removal. The few methods that kept the annotations were the only ones that were reported by PHPStan (I don't know why).